eZ/Bundle/EzPublishCoreBundle/Resources/config/papi.yml

```
@@ -16,6 +16,7 @@ services:
             - "@ezpublish.config.resolver"
             - "%ezpublish.api.inner_repository.class%"
             - "%ezpublish.api.role.policy_map%"
+            - '@eZ\Bundle\EzPublishCoreBundle\SiteAccess\LanguageResolver'
```

Let the API decide what to inject:
```suggestion
            - '@eZ\Publish\API\Repository\LanguageResolver'
```